softee220/naverent_crawling.py

```
@@ -0,0 +1,178 @@
+# coding: utf-8
+
+# # 웹에서 크롤링한 데이터를 모아서 본인이 직접 FAQ를 생성해보기
+
+# ### 1단계: Selenium으로 웹페이지 로드
+# 
+# Selenium을 사용하여 Yanolja 리뷰 페이지를 로드하고, 스크롤을 내려서 더 많은 데이터를 가져옵니다.
+
+# In[1]:
+
+
+get_ipython().system('pip install selenium')
```

Ruff (0.13.2)

Undefined name `get_ipython`

(F821)